doc/user-guide/user-guide.rst

```diff
@@ -76,38 +64,117 @@ At the same time, all images are annotated with the version information about th
 
 
-.. _Building-a-new-Cycle-or-Revision:
+.. _Building-a-New-Cycle-or-Revision:
 
-Building a new Cycle or Revision
+Creating a New Cycle or Revision
 ================================
 
-There are two ways users can run a build; manually on a private server or using the `TSSW Jenkins server`_.
-Since the build requires credentials for the nexus servers, both to access the licensed version of OpenSplice and to pull/push images to the nexus docker registry, it is advisable to use the `TSSW Jenkins server`_.
+The following is the procedure used to create and build a new cycle or revision.
+
+.. _Building-a-new-Cycle-or-Revision-Workflow:
+
+Workflow
+--------
+
+In general, it is expected that new cycle builds shall be deployed in one of the integration test stands to be tested before rolling to production (e.g. summit).
+Because the validation process can take some time and since summit environment is in constant need for updates (new revisions), a new cycle can only be considered officially released after the production rollout.
+
+To deal with this process in a more comprehensive way, this repository uses a different workflow than the  one adopted by `TSSW`_.
+
+.. _TSSW: https://tssw-developer.lsst.io/#id4
+
+  - For the cycle build we adopt ``master`` as the main branch, and also the branch that tracks what is running in the production environment.
+
+  - To start a new cycle, users must create a new branch that, initially, tracks the ``master`` branch.
+    The branch name must be named ``cycle/####``.
+    For instance, when starting cycle 21, users will create a branch ``cycle/0021`` that tracks ``master`` at the time.
+
+  - This branch should not receive any direct commit.
+    Instead, users should create a "ticket branch", with the name of the ticket they are working on.
+    Note that the repository is setup so all ``cycle/*`` branches are automatically protected against direct commits.
+
+    For instance, if the ticket you created to track the work to build cycle 21 is ``DM-12345``, you will create a ``tickets/DM-12345`` branch, based on the ``cycle/0021`` branch, to work with.
+
+    Once the work is completed, a PR must be issues to merge ``tickets/DM-12345`` to ``cycle/0021``.
 
-.. _TSSW Jenkins server: https://tssw-ci.lsst.org
+    .. attention::
+
+      Make sure you select the correct cycle branch when opening a PR.
+      By default github will assign the ``master`` branch as targets to all PRs.
+      You must manually select the cycle branch in this occasion.
+
+  - If new revisions to the current cycle (say cycle 20) is needed to support the production environment, those changes must be done on ticket branches and merged back to ``master``, as usual.
+
+  - If new revisions to the new cycle are needed, before the new cycle is officially released, they should create ticket branches out of the new cycle branch (e.g. ``cycle/0021``) and PRs must be opened and merged **back to the cycle branch**.
```

Comment:
change 'they should create ticket branches out of the new cycle branch' to 'ticket branches out of the new cycle branch should be created'